components/automate-ui/src/app/entities/teams/team.reducer.spec.ts

```diff
@@ -374,23 +374,23 @@ describe('teamStatusEntityReducer', () => {
     });
 
     const multipleState = genStateWithUserIDs(4);
-    const usersToRemoveMultiple = multipleState.userIDs.slice(0, 1);
     using([
-      [initialState, 'initial state with no inputs', 0, []],
-      [genArbitraryState(), 'non-initial state', 0, [faker.random.uuid()]],
-      [multipleState, 'when there are already user IDs', 4, usersToRemoveMultiple]
-    ], function (state: TeamEntityState, description: string,
-        _initialUserIdLength: number, usersToRemove: string[]) {
-      it('removes users from the payload to the state for ' + description, () => {
+      [initialState, 'for initial state with no inputs returns initial state', []],
+      [genArbitraryState(), 'for non-present user returns initial state', [faker.random.uuid()]],
+      [multipleState, 'removes single user from team',  multipleState.userIDs[0]],
+      [multipleState, 'removes multiple users from team',  multipleState.userIDs.slice(0, 2)],
+      [multipleState, 'removes all users from team',  multipleState.userIDs]
```

Comment:
Improve the tests relevant to this issue to have both deeper and broader coverage.